components/org.wso2.carbon.identity.api.server.configs/org.wso2.carbon.identity.api.server.configs.v1/src/gen/java/org/wso2/carbon/identity/api/server/configs/v1/model/ServerConfig.java

```diff
@@ -151,6 +154,32 @@ public ServerConfig addAuthenticatorsItem(AuthenticatorListItem authenticatorsIt
         return this;
     }
 
+        /**
```

Comment:
Shall we remove this comment if it is not required?

Reply:
since this is auto-generated, IMO we don't have to remove this.